.github/workflows/integration.yml

```diff
@@ -42,21 +42,28 @@ jobs:
           # install dxda
           sudo wget -O/usr/local/bin/dx-download-agent https://github.com/dnanexus/dxda/releases/download/v0.5.7/dx-download-agent-linux
           sudo chmod +x /usr/local/bin/dx-download-agent
-      - name: Run tests
+      - name: Run integration tests
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          AUTH_TOKEN: ${{ secrets.DX_STAGING_TOKEN }}
+          ROBOT_TOKEN: ${{ secrets.DX_STAGING_ROBOT_TOKEN }}
+          TEST_ROBOT_TOKEN: ${{ secrets.DX_STAGING_TEST_ROBOT_TOKEN }}
         run: |
           export PATH="$PATH:$HOME/.local/bin"
 
-          # set up DNAnexus staging environment
-          dx login --noprojects --staging --token $AUTH_TOKEN
+          # Set up DNAnexus staging environment, folder
+          dx login --noprojects --staging --token $ROBOT_TOKEN
           PROJECT=dxCompiler_playground
           dx select $PROJECT
-
-          # run tests
           cd ${GITHUB_WORKSPACE}
           FOLDER=/builds/test_gha_dxcompiler_$(date +%s)_$RANDOM
           dx mkdir $FOLDER
-          python3 scripts/run_tests.py --test M --folder $FOLDER --project $PROJECT
+
+          # Run tests
+          python3 scripts/run_tests.py --folder $FOLDER --project $PROJECT \
+          --test M
+          python3 scripts/run_multiuser_tests.py --folder $FOLDER --project $PROJECT \
+          --alice-token $ROBOT_TOKEN \
+          --bob-token $TEST_ROBOT_TOKEN
+
+          # TODO Also ensure folder is removed when tests fail
```

Comment:
It was a TODO (at least in release.yml) before this PR, and I don't make any change to that implementation here. I marked as TODO in both workflows because I don't see a difference in implementation.

Reply:
Changed to running each script with `--clean`, which should remove the folder in the finally block even if some tests failed.